lighthouse-cli/bin.ts

```diff
@@ -114,6 +117,7 @@ Example: --output-path=./lighthouse-results.html`
   .default('output', Printer.GetValidOutputOptions()[Printer.OutputMode.pretty])
   .default('output-path', 'stdout')
   .default('port', 9222)
+  .default('timeout', Driver.MAX_WAIT_FOR_FULLY_LOADED / 1000)
```

Comment:
are seconds the usual thing for command line timeouts? (obvs most of the js world tends towards ms)